optuna/integration/__init__.py

```diff
@@ -37,6 +37,7 @@
 if TYPE_CHECKING:
     from optuna.integration.allennlp import AllenNLPExecutor  # NOQA
     from optuna.integration.allennlp import AllenNLPPruningCallback  # NOQA
+    from optuna.integration.allennlp import dump_best_config  # NOQA
```

Comment:
Do we need to make this function public? I'm not sure but if this function is not intended by users, we should not add this function to the `optuna/integration/__init__.py`.

Reply:
Thank you, this import is not needed.

Reply:
8eede83

Reply:
Sorry, I changed my mind.
280763c  I restored `optuna.integration.allennlp` because there are examples using this namespace.
https://github.com/search?q=%22optuna.integration.allennlp%22+-repo%3Aoptuna%2Foptuna+-repo%3Aoptuna%2Fdoc-optuna-zh-cn&type=Code